.github/workflows/native-build-development.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: "Native Test - development"
+
+# Adding the dispatch event to allow restarting the build on demand
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+  repository_dispatch:
+
+jobs:
+  build_with_native:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: development
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Install JDK 8
+        uses: joschi/setup-jdk@v1.0.0
+        with:
+          java-version: 'openjdk8'
+
+      - name: Build Quarkus master
+        run: |
+          git clone https://github.com/quarkusio/quarkus.git \
+          && cd quarkus \
+          && mvn -B clean install -DskipTests -DskipITs
+
+      - name: Build Quickstart with native
+        run: |
+          mvn -B clean install -Pnative \
+            -Dquarkus.native.container-build=true \
+            -pl "!software-transactional-memory-quickstart"
```

Comment:
Is this the one that has a bug for the time being?

Reply:
yes (STM)

Reply:
Thanks

Reply:
I merged the PR that is supposed to fix it so we should be able to include it now.